packages/ps/src/commands/ps/wait.ts

```diff
@@ -17,10 +17,11 @@ export default class Wait extends Command {
       char: 'w',
       description: 'how frequently to poll in seconds (to avoid hitting Heroku API rate limits)',
       parse: async input => {
-        const w = parseInt(input, 10)
+        const w = Number.parseInt(input, 10)
```

Comment:
Why was the Number object used rather than leaving it as is?

Reply:
This is to satisfy a newer unicorn linting rule: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-number-properties.md